### PR TITLE
Fix a few warnings

### PR DESCRIPTION
**Pull request by neheb**

### CODE REVIEW

**Comment by neheb** [comment]
One benefit is that the dependency on gnu99 can now be removed.

**Comment by aircrack-ng** [comment]
It currently breaks compilation

**Comment by neheb** [comment]
Yeah the second commit is faulty. I'll remove it soon.

**Comment by aircrack-ng** [comment]
Still breaks on Travis and AppVeyor

File: src/aircrack-util/crypto_engine_loader.c [hunk 1]
```diff
@@ -198,7 +198,7 @@ void (*dso_ac_crypto_engine_calc_mic)(ac_crypto_engine_t * engine,
 	= NULL;
 #endif
 
-#if defined(CYGWIN)
+#if defined(__CYGWIN__)
```
Comment (jbenden) [inline src/aircrack-util/crypto_engine_loader.c]: This macro is defined via the Autoconf scripts.

**Review by jbenden — COMMENTED** [review]
Uck!

File: src/aircrack-util/crypto_engine_loader.c (hunk 1 above)
Comment (neheb) [reply src/aircrack-util/crypto_engine_loader.c]: Yeah it was an attempt to fix the buildbots. Apparently the reason they fail is that they pass -Werror, which is fairly amusing.

Speaking of which, the cygwin defines are all over the place.

Comment (jbenden) [reply src/aircrack-util/crypto_engine_loader.c]: Glad you like that. :)

**Review by jbenden — CHANGES_REQUESTED** [review]
Please look at the quantity of files/lines changed with the diff. I'm sure it broke Windows line endings on certain files...

Could you revert to an older commit and try that change over?

**Comment by neheb** [comment]
Yeah I'll fix it. I accidentally added unrelated stuff.

**Comment by neheb** [comment]
Rebased. Single buildbot failure seems to be due to some license thing.

**Review by jbenden — APPROVED** [review]

**Pull request merged**